frontend/packages/ceph-storage-plugin/integration-tests/tests/storage-dashboard.scenario.ts

```diff
@@ -0,0 +1,34 @@
+import { browser } from 'protractor';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import { dashboardIsLoaded } from '../../../console-shared/src/test-views/dashboard-shared.view';
+import { serviceName, clusterHealth, clusterName, allNodes } from '../views/storage-dashboard.view';
+import { execSync } from 'child_process';
+
+describe('Check data on Storage Dashboard.', () => {
+  beforeAll(async () => {
+    await browser.get(`${appHost}/dashboards/persistent-storage`);
+    await dashboardIsLoaded();
+  });
+
+  it('Check cluster health is OK', async () => {
+    expect(clusterHealth.getText()).toContain('is healthy');
```

Comment:
Do we really need `async` function here as we are not using `await` inside this test?

Reply:
Fixed